microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -348,13 +345,16 @@ To use the Fixed Version distribution mode:
       icacls {Fixed Version path} /grant *S-1-15-2-1:(OI)(CI)(RX)
       ```
 
-      PlayReady should be working now on the user's device.
-
-   1. To confirm that PlayReady is installed correctly, in the **Security** tab of the **Fixed Version** folder, make sure permissions are granted for `ALL APPLICATION PACKAGES` and `ALL RESTRICTED APPLICATION PACKAGES`, as shown below:
+   1. Confirm that permissions are granted for `ALL APPLICATION PACKAGES` and `ALL RESTRICTED APPLICATION PACKAGES` in the **Security** tab of the **Fixed Version** folder, as shown below:
```

Comment:
note:
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/can-may -- "Don't use _may_, which might be interpreted as providing permission"
repo uses 'may' in some places, but usually not w/ the implication that we are granting the reader permission 

here, sidestep can vs may, by standard direct instruction verb